spec/models/payment_spec.rb

```diff
@@ -44,4 +44,46 @@
       end
     end
   end
+
+  describe "validations of claims’ personal details" do
+    subject { build(:payment, claims: claims) }
+    let(:claims) do
+      build_list(:claim, 2, :approved,
+        national_insurance_number: "JM603818B",
+        teacher_reference_number: "1234567",
+        email_address: "email@example.com",
+        bank_sort_code: "112233",
+        bank_account_number: "95928482",
+        address_line_1: "64 West Lane",
+        student_loan_plan: StudentLoan::PLAN_1)
+    end
+
+    it "is valid when claims’ personal details match" do
+      expect(subject).to be_valid
+    end
+
+    it "is invalid when claims’ National Insurance numbers do not match" do
+      claims[0].national_insurance_number = "JM102019D"
+
+      expect(subject).not_to be_valid
+    end
+
+    it "is invalid when claims’ bank details do not match" do
+      claims[0].bank_account_number = "34192192"
```

Comment:
I feel like we're missing a bunch of coverage here for sort code, role number, etc.

Reply:
I'd thought it was perhaps enough to just have some specs for a few representative examples of attributes. I've added those two in now.